SA3/qa_web/templates/qa_web/EditUserProfile.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Edit User Profile</title>
+    </br>
+    <h1> Edit User Profile</h1>
+
+</head>
+<body>
+
+<!–– Django supported form is looped below to obtain the information -->
```

Comment:
Comments in templates can use the `{# this is a comment #}` syntax so they are not seen on the client side.

Reply:
removed, was no purpose other than comment purpose anyways.